src/scripts/get-binary-url.py

```diff
@@ -57,6 +60,23 @@ def main(browser_name, channel, application, os_name, bucket_name):
             source_url = locate_chrome(channel)
         elif browser_name == 'safari':
             source_url = locate_safari(channel)
+
+            # Version 83 of Safari Technology Preview was the last to be
+            # published with support for the "High Sierra" release of macOS
+            # [1]. That version also includes a regression which interrupts
+            # testing. In order to collect results from a High Sierra system,
+            # STP version 62 (as previously mirrored in this project's object
```

Comment:
```suggestion
            # STP version 82 (as previously mirrored in this project's object
```

Reply:
Oh, [that makes twice!](https://github.com/web-platform-tests/wpt/issues/17186#issuecomment-499164992) I really hope the number "82" doesn't show up on a bill or anything.

Thanks for the review!